src/rdo.rs

```diff
@@ -525,9 +525,13 @@ pub fn rdo_mode_decision<T: Pixel>(
       let ref_slot = ref_slot_set[i] as usize;
       let cmv = pmvs[ref_slot].unwrap();
 
-
       let b_me = motion_estimation(fi, fs, bsize, bo, ref_frames[0], cmv, pmv);
 
+      if !fi.config.speed_settings.encode_bottomup &&
+        (bsize == BlockSize::BLOCK_32X32 || bsize == BlockSize::BLOCK_64X64) {
```

Comment:
Couldn't you just update them unconditionally?

Reply:
I guess not because, if I understand well, pmvs are only computed and stored for 64x64 and 32x32 blocks.

Reply:
Right, we can but better not doing that. Because, for ex 32x32 has its childrenpartitions (ex: four of 16x16 blocks) split from it during topdown partition search, then if one of 16x16 MV update parent 32x32's pmv then other children may not have better pmv. This case would happen when 32x32 block straddle on object boundary, where 16x16 blocks in it have different MVs.